_episodes/05-lists.md

```diff
@@ -179,8 +179,10 @@ does not.
 >
 > Here is a visual example of how indexing a list of lists `x` works:
 >
-> [![The first element of a list.
-> Adapted from @hadleywickham.](../fig/indexing_lists_python.png)][hadleywickham-tweet]
+> [![x is represented as a pepper shaker containing several packets of pepper. [x[0]] is represented
+> as a pepper shaker containing a single packet of . xpepper[0] is represented as a single packet of
+> pepper. x[0][0] is represented as single fragment of pepper spice.  Adapted 
+> from @hadleywickham.](../fig/indexing_lists_python.png)][hadleywickham-tweet]
```

Comment:
Thanks @neil-csiro . I think this is a good representation of the image. There is a typo that needs to be fixed where it says `. xpepper[0] `. Also, I would change the wording of "as single fragment of pepper spice" to something like "as a single grain of pepper".

Reply:
Hi @ldko. I've made a new commit that I believe has corrected the typo and I've used a less peculiar phrasing for a unit of pepper. Thanks.